pkg/kubelet/container/runtime.go

```diff
@@ -424,6 +429,8 @@ type RunContainerOptions struct {
 	Devices []DeviceInfo
 	// The port mappings for the containers.
 	PortMappings []PortMapping
+	// The annotations for the container
+	Annotations []Annotation
```

Comment:
Expand the comment a bit more to explain what type of annotations these are, e.g., 
```
Additional annotations for the container to pass to the runtime.
These annotations are generated by other components (i.e.,
not users). Currently, only device plugins populate the annotations.
```

Reply:
@RenaudWasTaken this still needs to be taken care of.